src/content/docs/en/guides/monitor/sentry.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Monitor your Astro Site with Sentry
+description: How to monitor your Astro site with Sentry
+type: recipe
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+As Astro's [Official Monitoring Partner](https://astro.build/blog/sentry-official-monitoring-partner/), Sentry offers a comprehensive application monitoring and error tracking service designed to help developers identify, diagnose, and resolve issues in real-time.
+
+Sentry's Astro SDK enables automatic reporting of errors and tracing data in your Astro application.
+
+## Project Configuration
+
+A full list of pre-requisites can be found in Sentry Astro docs [here](https://docs.sentry.io/platforms/javascript/guides/astro/#prerequisites).
+
+### Install, Configure and Verify
+
+Sentry captures data by using an SDK within your application’s runtime.
+
+<Steps>
+1. Install the SDK by using the astro CLI:
```

Comment:
```suggestion
### Install

Sentry captures data by using an SDK within your application’s runtime.

Install the SDK by running the following command for the package manager of your choice in the Astro CLI:
```
That was a lot to have in one heading, so instead of steps, I suggest making these three separate sections. Bonus, they show up as separate entries in the Table of Contents in the "on this page" sidebar, and each section is easier to notice, and link to separately.